api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -337,6 +343,16 @@ private void validateUpdatedProfile(Profile updatedProfile, Profile prevProfile)
       // See RW-1488.
       throw new BadRequestException("Changing username is not supported");
     }
+    if (updatedProfile.getVerifiedInstitutionalAffiliation() == null
+        && prevProfile.getVerifiedInstitutionalAffiliation() != null) {
+      throw new BadRequestException("Cannot delete Verified Institutional Affiliation.");
+    }
+
+    if (!prevProfile.getGivenName().equalsIgnoreCase(updatedProfile.getGivenName())
```

Comment:
moved from updateProfileForUser to make this common between update and adminUpdate

Planning to make this more explicit as part of RW-5104